src/main/java/hnrc/example/EventPublisher.java

```diff
@@ -13,30 +12,29 @@
 public class EventPublisher {
```

Comment:
All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.